Challenge_2/Python/Papersquares/Challenge2uniquevalue.py

```diff
@@ -0,0 +1,8 @@
+
+list = [2,3,4,2,3,5,4,6,4,6,9,'a',10,9,8,7,8,10,7]
+
+newlist = []
```

Comment:
What benefit does this have in regards to things?  Just because it is shorter code it makes this run faster or does it have other functionality?